statsmodels/stats/anova.py

```diff
@@ -333,19 +331,9 @@ def anova_lm(*args, **kwargs):
         raise ValueError("Multiple models only supported for type I. "
                          "Got type %s" % str(typ))
 
-    ### COMPUTE Anova TYPE I ###
-
-    # if given a single model
-    if len(args) == 1:
-        return anova_single(*args, **kwargs)
```

Comment:
why did you remove this?

Reply:
That is duplicate code, also a couple of lines above the diff.

Reply:
I see now, looks good